flow/e2e/mongo/mongo_test.go

```diff
@@ -648,19 +648,15 @@ func (s MongoClickhouseSuite) Test_Json_Types() {
 		{Key: "neg_int64", Value: int64(-9223372036854775807)},
 
 		// Floating point types
-		{Key: "float32", Value: float32(3.14)},
 		{Key: "float64", Value: float64(3.14159265359)},
-		{Key: "neg_float32", Value: float32(-3.14)},
 		{Key: "neg_float64", Value: float64(-3.14159265359)},
+		{Key: "max_float64", Value: float64(math.MaxInt64)},
+		{Key: "min_float64", Value: float64(math.MinInt64)},
```

Comment:
If the goal was to also test that the e2e doesn't crash on outside-of-int64 integers on CH side, maybe add a +-2^65 or something like that (via strconv.ParseFloat())?